src/sidebar/oauth-auth.js

```diff
@@ -66,34 +65,6 @@ function auth($http, $rootScope, $window,
     );
   }
 
-  /**
-   * Return a new TokenInfo object from the given tokenUrl endpoint response.
-   * @param {Object} response - The HTTP response from a POST to the tokenUrl
-   *                            endpoint (an Angular $http response object).
-   * @returns {TokenInfo}
-   */
-  function tokenInfoFrom(response) {
-    var data = response.data;
-    return {
-      accessToken:  data.access_token,
-
-      // Set the expiry date to some time slightly before that implied by
-      // `expires_in` to account for the delay in the client receiving the
-      // response.
-      expiresAt: Date.now() + ((data.expires_in - 10) * 1000),
-
-      refreshToken: data.refresh_token,
-    };
-  }
-
-  function formPost(url, data) {
-    data = queryString.stringify(data);
-    var requestConfig = {
-      headers: {'Content-Type': 'application/x-www-form-urlencoded'},
-    };
-    return $http.post(url, data, requestConfig);
-  }
-
```

Comment:
`tokenInfoFrom`, `formPost`, `exchangeJWT` and `exchangeAuthCode` have all been moved into the `OAuthClient` class.